firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp

```diff
@@ -209,7 +221,8 @@ void stm32_standby() {
 
 	// Wait for event - this should never return as it kills the chip until a reset
 	__WFE();
+	__WFE();
 
 	// Lastly, reboot
-	NVIC_SystemReset();
+	//NVIC_SystemReset();
```

Comment:
if we do not need something simply remote it

Reply:
this looks like unrelated change that snuck in, it's in the next function

Reply:
can we please remove standby already? do we know enough to know that we do not need it?

Reply:
Not yet. I want to finish testing current consumption in both modes, and see what would work. I do believe we can probably dump standby, but please wait for confirmation.